### PR TITLE
Added automatic triggering if we have got through the feedstock creation process.

### PR DESCRIPTION
**Pull request by pelson**

### CODE REVIEW

**Comment by pelson** [comment]
Ah, such a shame!

```
./.CI/create_feedstocks: line 15: ./trigger_travis_conda-forge.github.io.sh: No such file or directory
```

File: .CI/trigger_travis_conda-forge.github.io.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
+"""
+Trigger the conda-forge.github.io Travis job to restart.
+
+"""
+import requests
+import six
```
Comment (jakirkham) [inline .CI/trigger_travis_conda-forge.github.io.sh]: Is this `.sh`?

**Comment by jakirkham** [comment]
Looks like you have a Python file in there but it claims to be bash/sh.

File: .CI/trigger_travis_conda-forge.github.io.sh (hunk 1 above)
Comment (pelson) [inline .CI/trigger_travis_conda-forge.github.io.sh]: Ah :hankey: - not enough sleep clearly. Thanks @jakirkham. I'll submit a third PR :disappointed:

File: .CI/trigger_travis_conda-forge.github.io.sh [hunk 2]
```diff
@@ -0,0 +1,39 @@
+"""
```
Comment (jakirkham) [inline .CI/trigger_travis_conda-forge.github.io.sh]: Maybe also `#!/usr/bin/env python`.

File: .CI/trigger_travis_conda-forge.github.io.sh (hunk 1 above)
Comment (jakirkham) [inline .CI/trigger_travis_conda-forge.github.io.sh]: I love how they give that emoticon eyes. No worries. I'll try to type really fast this time. :wink:

File: .CI/trigger_travis_conda-forge.github.io.sh (hunk 2 above)
Comment (pelson) [inline .CI/trigger_travis_conda-forge.github.io.sh]: I just executed it in #187. :+1:

Comment (jakirkham) [inline .CI/trigger_travis_conda-forge.github.io.sh]: Yep, I saw it. That change looked good.

**Comment by pelson** [comment]
It was worth it - I've just seen conda-forge.github.io get triggered. :tada:

**Comment by jakirkham** [comment]
So does it just restart the same build over and over?

**Comment by jakirkham** [comment]
Haven't read it all yet.

**Comment by pelson** [comment]
> So does it just restart the same build over and over?

Yes. It essentially clicks the restart button on travis-ci which is all @ocefpaf and I have been doing to trigger an update to the docs/feedstocks repo.

**Comment by jakirkham** [comment]
Ok, so, I withdraw my worry about collisions then.

**Comment by ocefpaf** [comment]
> Yes. It essentially clicks the restart button on travis-ci which is all @ocefpaf and I have been doing to trigger an update to the docs/feedstocks repo.

And I loose my job (again) to smart script :sob:

**Comment by pelson** [comment]
> And I loose my job (again) to smart script :sob:

Focus on recipes, and I'll automate the rest :wink:

**Comment by jakirkham** [comment]
I definitely feel like the more recipes coming through here and getting added the more I want things automated not less. Just so we don't saturate our bandwidth.

**Pull request merged**